client/js/screens/CampaignModal/CampaignModal.js

```diff
@@ -1,8 +1,92 @@
 import React from 'react';
-import {Text} from 'react-native';
+import {Text, View, ScrollView, Image, Linking} from 'react-native';
+import {withNavigation} from 'react-navigation';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+import styles from '../Campaign/styles';
+import {colors} from '../../config/styles';
+// import SafeAreaView from 'react-native-safe-area-view';
```

Comment:
Please remove commented code if not needed.

Reply:
Done